sonarts-core/tests/rules/boolParamDefaultRule/boolParamDefaultRule.lint.ts

```diff
@@ -0,0 +1,12 @@
+function f1(flag?: boolean) {}
+//          ^^^^^^^^^^^^^^{{Provide default value for this parameter.}}
```

Comment:
Provide a default value for 'flag' so that the logic of the function is more evident when this parameter is missing.